packages/openai/src/responses/openai-responses-language-model.ts

```diff
@@ -1106,6 +1242,28 @@ function isErrorChunk(
   return chunk.type === 'error';
 }
 
+type OpenAIImagePartialChunk = {
+  type: 'response.image_generation_call.partial_image';
+  sequence_number: number;
+  output_index: number;
+  item_id: string;
+  partial_image_index: number;
+  partial_image_b64: string;
+  size?: string;
+  quality?: string;
+  background?: string;
+  output_format?: string;
+};
```

Comment:
should this be part of the response schema so the structure is validated?

Reply:
removed for now